.github/workflows/gui.yml

```
@@ -0,0 +1,32 @@
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    # 检查出代码
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    # 设置.NET环境
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
```

actionlint (1.7.7)

the runner of "actions/setup-dotnet@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)